[2024-01-26T20:59:26.520+0000] {processor.py:153} INFO - Started process (PID=37844) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:26.522+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T20:59:26.523+0000] {logging_mixin.py:137} INFO - [2024-01-26T20:59:26.523+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:43.881+0000] {logging_mixin.py:137} INFO - [2024-01-26T20:59:43.873+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_scripts/etl_dim_departamento.py", line 11, in <module>
    'user': config['Database']['user'],
  File "/usr/local/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'Database'
[2024-01-26T20:59:43.886+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:43.917+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 17.402 seconds
[2024-01-26T20:59:58.202+0000] {processor.py:153} INFO - Started process (PID=37897) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:58.203+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T20:59:58.205+0000] {logging_mixin.py:137} INFO - [2024-01-26T20:59:58.204+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:58.918+0000] {logging_mixin.py:137} INFO - [2024-01-26T20:59:58.908+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_scripts/etl_dim_departamento.py", line 11, in <module>
    'user': config['Database']['user'],
  File "/usr/local/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'Database'
[2024-01-26T20:59:58.920+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T20:59:58.947+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.750 seconds
[2024-01-26T21:00:08.138+0000] {processor.py:153} INFO - Started process (PID=37909) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:08.140+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:00:08.143+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:08.142+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:08.817+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:08.811+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_scripts/etl_dim_departamento.py", line 11, in <module>
    'user': config['Database']['user'],
  File "/usr/local/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'Database'
[2024-01-26T21:00:08.820+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:08.850+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.717 seconds
[2024-01-26T21:00:39.984+0000] {processor.py:153} INFO - Started process (PID=37955) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:39.987+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:00:39.989+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:39.989+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:41.456+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:41.448+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_scripts/etl_dim_departamento.py", line 11, in <module>
    'user': config['Database']['user'],
  File "/usr/local/lib/python3.7/configparser.py", line 958, in __getitem__
    raise KeyError(key)
KeyError: 'Database'
[2024-01-26T21:00:41.460+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:41.490+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 1.512 seconds
[2024-01-26T21:00:59.810+0000] {processor.py:153} INFO - Started process (PID=37975) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:59.811+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:00:59.812+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:59.812+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:59.834+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:00:59.833+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:00:59.834+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:00:59.857+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.050 seconds
[2024-01-26T21:01:25.393+0000] {processor.py:153} INFO - Started process (PID=38010) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:25.394+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:01:25.395+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:01:25.395+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:25.416+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:01:25.416+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:01:25.416+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:25.438+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.048 seconds
[2024-01-26T21:01:55.818+0000] {processor.py:153} INFO - Started process (PID=38062) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:55.819+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:01:55.820+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:01:55.820+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:55.833+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:01:55.832+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:01:55.834+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:01:55.858+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.045 seconds
[2024-01-26T21:02:26.511+0000] {processor.py:153} INFO - Started process (PID=38098) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:26.512+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:02:26.513+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:02:26.513+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:26.521+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:02:26.521+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:02:26.521+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:26.536+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.028 seconds
[2024-01-26T21:02:57.053+0000] {processor.py:153} INFO - Started process (PID=38134) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:57.141+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:02:57.143+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:02:57.143+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:57.162+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:02:57.162+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:02:57.163+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:02:57.197+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.149 seconds
[2024-01-26T21:03:27.862+0000] {processor.py:153} INFO - Started process (PID=38179) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:03:27.863+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:03:27.865+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:03:27.864+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:03:27.875+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:03:27.875+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:03:27.876+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:03:27.892+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.033 seconds
[2024-01-26T21:03:59.842+0000] {processor.py:153} INFO - Started process (PID=38215) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:03:59.908+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:03:59.934+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:03:59.934+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:04:00.105+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:04:00.105+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:04:00.106+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:04:00.575+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 0.736 seconds
[2024-01-26T21:04:08.562+0000] {processor.py:153} INFO - Started process (PID=38224) to work on /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:04:08.827+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_scripts/etl_dim_departamento.py for tasks to queue
[2024-01-26T21:04:08.828+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:04:08.828+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:04:08.843+0000] {logging_mixin.py:137} INFO - [2024-01-26T21:04:08.842+0000] {dagbag.py:320} INFO - File /opt/airflow/dags/etl_scripts/etl_dim_departamento.py assumed to contain no DAGs. Skipping.
[2024-01-26T21:04:08.843+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_scripts/etl_dim_departamento.py
[2024-01-26T21:04:09.056+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_scripts/etl_dim_departamento.py took 1.214 seconds
